shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -541,12 +542,15 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
       return null;
     }
 
+    // Generate accessibility node for platform views using a virtual display.
```

Comment:
nit: I think a future reader may find it a little confusing that there's more logic below to deal with platform view nodes, depending on the composition mechanism. I'd suggest elaborating a little more in this comment, mentioning explicitly that for HC the platform view is added when it's parent semantic node is processed below (I'd also leave a comment below pointing up here for the VD case).

Reply:
Added more background information.